CHANGES.md

```diff
@@ -6,6 +6,7 @@ Changes
 
 ### v4.1-r2
 
+* 2023-02-23 - Feature: Allow right-side blocks drawer to be extended by default #169.
```

Comment:
```suggestion
### Unreleased

* 2023-02-23 - Feature: Allow right-side blocks drawer to be extended by default #169.

### v4.1-r2
```

This is not completely obvious and documented in our wiki yet, but you should not add new features under headings of published releases.